doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -1,4 +1,10 @@
-\section{Group -- Advanced Surface Concepts}\label{group-advanced-surface-concepts}
+\documentclass[10pt]{article}
```

Comment:
@Myoldmopar No idea how this got in there.  I simply edited the text on line 232/238 and nothing else.  I will eliminate these lines and the one at the end and hope they don't magically pop up again.